etcdserver/auth/auth_test.go

```diff
@@ -73,14 +88,14 @@ func TestMergeUser(t *testing.T) {
 		},
 		{
 			User{User: "foo"},
-			User{User: "foo", Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
```

Comment:
that is ok. but the original test case is to ensure non-empty string password will not be overwrite. you change changes this test cases completely.